derive_builder/src/options/struct_mode.rs

```diff
@@ -49,6 +57,25 @@ impl OptionsBuilderMode for StructMode {
         self.builder_name(value.clone());
     }
 
+    fn parse_derive(&mut self, nested: &[syn::NestedMetaItem]) {
+        let mut traits = vec![];
+        for x in nested {
+            match *x {
+                // We don't allow name-value pairs or further nesting here, so
+                // only look for words.
+                syn::NestedMetaItem::MetaItem(syn::MetaItem::Word(ref tr)) => {
+                    traits.push(tr.clone())
```

Comment:
I've added a warning at the location in the code review. At the moment it also suppresses forwarding the trait, but I can remove that so we push the warning and then let compilation fail.